task/net/src/bsp_support.rs

```diff
@@ -14,6 +14,7 @@
 //! A BSP module is expected to export a single type, called `BspImpl`, which
 //! implements the `Bsp` trait from this module.
 
+use crate::notifications;
```

Comment:
Github actions points out this is unused in several build configs. Maybe move into the cfg_if below?

Reply:
good catch, done